test/catalog-fixtures/DefaultCatalogFixture.json

```diff
@@ -16,6 +16,14 @@
 			"name": "Best Practices",
 			"engine": "pmd"
 		},
+		{
```

Comment:
This goes along with a change to RuleManager.test.ts, the test was previously including the `Design` category in `Filtering by multiple categories runs any rule in either category` but `Design` didn't actually match any rules. This adds some rules that match the test. I included ecmascript in the pmd engine for consistency when we add support back, it doesn't negatively affect the tests.